.storybook-s2/docs/Release Notes.mdx

```diff
@@ -4,6 +4,42 @@ export default MDXLayout;
 
 # Release Notes
 
+## v0.9.0
+
+### Updates
+
+* [Combobox](?path=/docs/combobox--docs): Support for virtualization and async loading
+* [Dialog](?path=/docs/picker--dialog): Update sizes
+* [Picker](?path=/docs/picker--docs): Support for virtualization and async loading
+* [TableView](?path=/docs/tableview--docs): Support custom menu's in columns
+* `dist` was renamed to `disc` (fixed spelling) in spectrum theme for `listStyleType`
```

Comment:
just a reminder to remove this when we bring in the copy from the canvas since it will be under the theme update section